WEB-INF/classes/uon/seng2050/assignment/model/Issue.java

```diff
@@ -0,0 +1,123 @@
+package uon.seng2050.assignment.model;
+
+import io.seanbailey.adapter.Model;
+import java.util.Date;
+import java.util.UUID;
+
+public class Issue extends Model {
+
+  private UUID id;
+  private Category category;
+  private SubCategory subCategory;
+  private State state;
+  private String title;
+  private String body;
+  private Date createdAt;
+  private Date updatedAt;
+  private boolean locked;
+  //List for comments?
+
+  public Issue() {
+    //constructor
+  }
+
+  @Override
+  public boolean validate() {
+    boolean isValid = true;
+    //run data checks
+    if(isValid) {
+      return true;
+    }
+    else {
+      addError("Invalid Issue data");
+    }
+    return false;
+  }
+
+  public enum State {
+    NEW, INPROGRESS, COMPLETED, RESOLVED
+  }
+
+  public enum Category {
+    Network, Software, Hardware, Email, Account, Other
+  }
```

Comment:
enum values need to be uppercase, and snake case. So `SlowToLoad` should be `SLOW_TO_LOAD`